osh.spec

```diff
@@ -175,7 +176,10 @@ rm -rf %{buildroot}%{python3_sitelib}/scripts
 %attr(755,root,root) %{_bindir}/osh-cli
 %{_bindir}/covscan
 %attr(644,root,root) %config(noreplace) /etc/osh/client.conf
+%dir %{bash_completions_dir}
 %{bash_completions_dir}/osh-cli.bash
+%dir %{zsh_completions_dir}
+%{zsh_completions_dir}/_osh-cli
```

Comment:
As noted on the [original review](https://gitlab.cee.redhat.com/covscan/covscan/-/merge_requests/398#note_6865254), this can be simplified as:
```
%{bash_completions_dir}
%{zsh_completions_dir}
```

`rpmbuild` will pick the whole subtrees, each consisting of one directory and one regular file.

Reply:
Addressed in 77ffee2 and a97c8c5.